crates/vsvg/src/document/mod.rs

```diff
@@ -78,7 +78,7 @@ pub trait DocumentTrait<L: LayerTrait<P, D>, P: PathTrait<D>, D: PathDataTrait>:
 
         let doc = document_to_svg_doc(self);
         let mut svg = String::new();
-        write!(svg, "{doc}").map(|_| svg)
+        write!(svg, "{doc}").map(|()| svg)
```

Comment:
Thanks for fixing those (it's a new lint introduce in 1.73, released yesterday).